Lib/defcon/objects/imageSet.py

```diff
@@ -222,14 +223,14 @@ def makeFileName(self, fileName):
         """
         Make a file system legal version of **fileName**.
         """
-        if not isinstance(fileName, str):
-            fileName = str(fileName)
+        if not isinstance(fileName, unicode):
+            fileName = unicode(fileName)
```

Comment:
I'm not sure the isinstance is even needed here since doing `unicode(u"foo")` doesnt create a new string (since they're immutable).